core/src/main/java/com/datastax/oss/driver/internal/core/metadata/schema/parsing/RelationParser.java

```diff
@@ -83,6 +87,26 @@ protected RelationParser(SchemaRows rows, InternalDriverContext context) {
     return builder.build();
   }
 
+  /**
+   * Handle a few oddities in Cassandra 4: some options still appear in system_schema.tables, but
+   * they are not valid in CREATE statements anymore. We need to exclude them from our metadata,
+   * otherwise {@link Describable#describe(boolean)} will generate invalid CQL.
```

Comment:
I'm not sure if this is normal, maybe we should open a Cassandra ticket to remove those options from the system tables.

Reply:
[CASSANDRA-15604](https://issues.apache.org/jira/browse/CASSANDRA-15604)

Reply:
After some discussion, we agreed to not fix the C* ticket, so we'll keep this workaround.